prep_inputs_for_combiner.py

```diff
@@ -0,0 +1,301 @@
+        os.path.basename(fp).replace(f'.{file_suffix}', ''): fp
+        for fp in found_files
+    }
+    return found_path_by_sname
+
+
+def _randomise_pop_labels(sample_df):
+    # Adding population labels for 2/3 of the samples in a group;
+    # the rest will be used to test the ancestry inferring methods
+    return list(
+        sample_df\
+            .groupby(['Population']) \
+            .apply(lambda x: x.iloc[:int(x['Population'].size / 1.5)]) \
+            ['Individual.ID']
+    )
+
+
+def _move_locally(gvcf_by_sample, dataset, picard_file_by_sname_by_key):
+    local_gvcf_by_sample = dict()
+    local_picard_file_by_sname_by_key = defaultdict(dict)
+    for sample, gvcf_path in gvcf_by_sample.items():
+        local_path = f'gs://cpg-fewgenomes-upload/{sample}/gvcf/{basename(gvcf_path)}'
+        if not file_exists(local_path):
+            run_cmd(f'gsutil cp {gvcf_path} {local_path}')
+        if not file_exists(local_path + '.tbi'):
+            run_cmd(f'gsutil cp {gvcf_path}.tbi {local_path}.tbi')
+        local_gvcf_by_sample[sample] = local_path
+
+        for picard_key, picard_path_by_sname in picard_file_by_sname_by_key.items():
+            picard_path = picard_path_by_sname.get(sample)
+            if picard_path:
+                local_path = f'gs://cpg-fewgenomes-upload/{sample}/picard_files/' \
+                             f'{basename(picard_path)}'
+                if not file_exists(local_path):
+                    run_cmd(f'gsutil cp {picard_path} {local_path}')
+                local_picard_file_by_sname_by_key[picard_key][sample] = local_path
+    return local_gvcf_by_sample, local_picard_file_by_sname_by_key
+
+
+@click.command()
+@click.option(
+    '--dataset',
+    'dataset_name',
+    required=True,
+    help='Dataset name, e.g. "50genomes". Assumes that '
+         '`{datasets_dir}/{datasets_name}/samples.ped` exists, unless --ped file is '
+         'specified explicitly.'
+)
+@click.option(
+    '--ped',
+    'samples_ped',
+    help='Ped file with input sample names. If not provided, '
+         '`{datasets_dir}/{datasets_name}/samples.ped` is read.'
+)
+@click.option(
+    '--warp-executions-bucket',
+    'warp_executions_bucket',
+    default=('gs://playground-us-central1/cromwell/executions/'
+             'WGSMultipleSamplesFromBam'),
+    help='Bucket with WARP workflow outputs'
+)
+@click.option(
+    '--datasets-dir',
+    'datasets_dir',
+    default='datasets',
+    help='Output folder. Default is "datasets/"'
+)
+@click.option(
+    '--work-dir',
+    'work_dir',
+    help='Directory to store temporary files'
+)
+@click.option(
+    '--split-rounds',
+    'split_rounds',
+    is_flag=True,
+    help='Break samples into 2 groups to produce tests for the gVCF combiner'
+)
+@click.option(
+    '--randomise-pop-labels',
+    'randomise_pop_labels',
+    is_flag=True,
+    help='Remove population labels for 1/3 of the samples to test sample-qc '
+         'ancestry detection'
```

Comment:
```suggestion
         'ancestry inference.'
```
Think inference is more accurate.